cypress/support/pageBuilder/pbCreateBlock.js

```diff
@@ -0,0 +1,45 @@
+import { GraphQLClient } from "graphql-request";
+
+Cypress.Commands.add("pbCreateBlock", (blockVariables, categorySlug) => {
+    cy.login().then(user => {
+        const client = new GraphQLClient(Cypress.env("GRAPHQL_API_URL"), {
+            headers: {
+                authorization: `Bearer ${user.idToken.jwtToken}`
+            }
+        });
+
+        const createBlockMutation = `
```

Comment:
```suggestion
        const createBlockMutation = /* GraphQL */`
```

By adding this hint, prettier will be able to format the GQL query nicely. Example:

```ts
const USER_PAT_FIELDS = /* GraphQL */ `
    fragment UserPatFields on UserPat {
        name
        meta
        token
        expiresOn
        user {
            email
        }
    }
`;
```

Reply:
I got some weird results with this